README.md

```diff
@@ -115,5 +110,9 @@ This project will create several things for you.  It will deploy an Azure Functi
 If you encounter issues running this demo, there are several ways you can debug what may be wrong.
 
 1. In *./JfkWebApiSkills/JfkInitializer/Program.cs*, you can set the "DebugMode" variable to true in order to see the output for REST calls that are made to create resources and why they may be failing.  The Initializer will prompt you to do this if it suspects there is an issue creating anything it needs to create.
-2. You can add the "enriched" field to your index to see detailed output of all skills that are run.  [Details on how to do this can be found here.](https://docs.microsoft.com/en-us/azure/search/cognitive-search-concept-troubleshooting#tip-4-looking-at-enriched-documents-under-the-hood)  You will be making this change in *./JfkWebApiSkills/JfkInitializer/SearchResources.cs*.
+2. You can add the "enriched" field to your index to see detailed output of all skills that are run.  [Details on how to do this can be found here.](https://docs.microsoft.com/azure/search/cognitive-search-concept-troubleshooting#tip-4-looking-at-enriched-documents-under-the-hood)  You will be making this change in *./JfkWebApiSkills/JfkInitializer/SearchResources.cs*.
 3. If all else fails, feel free to [post your problem as a Github issue](https://github.com/Microsoft/AzureSearch_JFK_Files/issues/new), which we monitor and respond to when necessary.
+
+## Next Steps
+
+The [Azure Architecture Center](https://aka.ms/architecture) has more content on [this solution](https://docs.microsoft.com/azure/architecture/solution-ideas/articles/cognitive-search-with-skillsets) and [others in the Cognitive Services & AI space](https://docs.microsoft.com/en-us/azure/architecture/data-guide/technology-choices/cognitive-services).
```

Comment:
@CoromT , do you think we should wait to see what this doc actually says before merging this (i.e. make sure it relays what our team wants)?  Also, I'm not sure if we are comfortable with the link to the additional cognitive services options, since that is pushing people away from using our product and towards using cognitive services standalone (obviously is fine and good for Microsoft overall, but not really the purpose of this sample).

Reply:
Content is now live: https://docs.microsoft.com/azure/architecture/solution-ideas/articles/cognitive-search-with-skillsets  -- we can remove anything else in this sentence you wish.  I was just padding it out to look a bit more substantial.

Reply:
Removed the trailing links.